test/unit/accessors.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { Expect, Test } from "alsatian";
+import * as util from "../src/util";
+
+export class AccessorTests {
+    @Test("get accessor")
+    public getAccessor(): void {
+        const code =
+            `class Foo {
+                get foo() { return "foo"; }
+            }
+            const f = new Foo();
+            return f.foo;`;
+        Expect(util.transpileAndExecute(code)).toBe("foo");
+    }
+
+    @Test("get accessor in base class")
+    public getAccessorInBaseClass(): void {
+        const code =
+            `class Foo {
+                get foo() { return "foo"; }
+            }
+            class Bar extends Foo {}
+            const b = new Bar();
+            return b.foo;`;
+        Expect(util.transpileAndExecute(code)).toBe("foo");
+    }
+
+    @Test("get accessor override")
+    public getAccessorOverride(): void {
+        const code =
+            `class Foo {
+                foo = "foo";
+            }
+            class Bar extends Foo {
+                get foo() { return "bar"; }
+            }
+            const b = new Bar();
+            return b.foo;`;
+        Expect(util.transpileAndExecute(code)).toBe("bar");
+    }
+
+    @Test("get accessor overridden")
+    public getAccessorOverridden(): void {
+        const code =
+            `class Foo {
+                get foo() { return "foo"; }
+            }
+            class Bar extends Foo {
```

Comment:
You have a test base with accessor, extension without, base without, extension with accessor. Could you also add one where they both have a get accessor?